Makefile

```diff
@@ -395,6 +395,7 @@ delete-all-minikube-profiles:
 
 # Current Operator version
 OPERATOR_VERSION ?= 0.0.1
+BUNDLE_OUTPUT_DIR := $(or ${BUNDLE_OUTPUT_DIR},./bundle)
```

Comment:
Use a nested directory inside `${BUILD_FOLDER}/bundle` to avoid more files in the root directory